ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,187 @@
+    if otel_value == "always_on" or otel_value == "parentbased_always_on":
+        return "1.0"
+    elif otel_value == "always_off" or otel_value == "parentbased_always_off":
+        return "0.0"
+    elif otel_value == "traceidratio" or otel_value == "parentbased_traceidratio":
+        return os.environ.get("OTEL_TRACES_SAMPLER_ARG", "1")
+    else:
+        return otel_value
+
+
+def _remap_traces_exporter(otel_value):
+    """Remaps the otel trace exporter to ddtrace trace enabled"""
+    if otel_value == "none":
+        return "False"
+    log.warning(
+        "A trace exporter value '%s' is set, but not supported. Traces will be exported to Datadog.", otel_value
+    )
+    return ""
+
+
+def _remap_metrics_exporter(otel_value):
+    """Remaps the otel metrics exporter to ddtrace metrics exporter"""
+    if otel_value != "none":
+        log.warning(
+            "An unrecognized runtime metrics exporter '%s' is being "
+            "used; setting DD_RUNTIME_METRICS_ENABLED to False.",
+            otel_value,
+        )
+    return "False"
+
+
+def _remap_logs_exporter(otel_value):
+    """Logs warning when OTEL Logs exporter is configured. DDTRACE does not support this configuration."""
+    if otel_value != "none":
+        log.warning(
+            "Unsupported OTEL logs exporter value detected: %s. Only the 'none' value is supported.", otel_value
+        )
+        return ""
+    return ""
+
+
+def _remap_otel_tags(otel_value):
+    """Remaps the otel tags to ddtrace tags"""
+    dd_tags = []
+    remaining_tags = []
+    otel_tags = otel_value.split(",")
+    otel_user_tag_dict = dict()
+
+    try:
+        for tag in otel_tags:
+            tag_pair = tag.split("=")
+            otel_user_tag_dict[tag_pair[0]] = tag_pair[1]
+
+        for otel_key, dd_key in OTEL_UNIFIED_TAG_MAPPINGS.items():
+            if otel_key in otel_user_tag_dict.keys():
+                dd_tags.append("{}:{}".format(dd_key, otel_user_tag_dict[otel_key]))
+
+        for key, value in otel_user_tag_dict.items():
+            if key.lower() not in OTEL_UNIFIED_TAG_MAPPINGS.keys():
+                if len(dd_tags) < 10:
+                    dd_tags.append("{}:{}".format(key, value))
+                else:
+                    remaining_tags.append("{}:{}".format(key, value))
+    except Exception:
+        log.warning("DDTRACE failed to read OTEL_RESOURCE_ATTRIBUTES. This value is misformatted: %s", otel_value)
+
+    if len(otel_user_tag_dict.items()) > 10:
+        log.warning(
+            "To preserve metrics cardinality, only the following first 10 tags have been processed %s. "
+            "The following tags were not ingested: %s",
+            dd_tags,
+            remaining_tags,
+        )
+    return ",".join(dd_tags)
+
+
+def _remap_otel_sdk_config(otel_value):
+    """Remaps the otel sdk config to ddtrace sdk config"""
+    if otel_value == "false":
+        return "True"
```

Comment:
can we get a comment here, I am not sure why false should mean true here?

Reply:
oh, is this because this is "OTEL_SDK_DISABLED" and we are remapping to an "ENABLED" config?